tutorials/sensor/lidar_node.cc

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#include <iostream>
+#include <string>
+#include <ignition/msgs.hh>
+#include <ignition/transport.hh>
```

Comment:
I think it's good to recommend that users only include what they will need. These files are including all of `ign-msgs` and `ign-transport`, which adds unnecessary overhead.

I think you could just include:

```
#include <ignition/msgs/twist.pb.h>
#include <ignition/msgs/laser_scan.pb.h>
#include <ignition/transport/Node.hh>
```